packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx

```diff
@@ -205,6 +207,12 @@ const TranslationInput = (props: TranslationInputProps) => {
                 <div>{props.warning}</div>
               </LeftColumn>
             );
+          if (props.additionalInfo)
+            return (
+              <LeftColumn>
+                <InfoMessage>{props.additionalInfo}</InfoMessage>
+              </LeftColumn>
+            );
```

Comment:
I wonder if this accommodates the following scenario
<img width="841" alt="Screenshot 2024-01-31 at 14 42 45" src="https://github.com/commercetools/ui-kit/assets/49066275/bc936587-4917-4289-94b3-d692a6a40ba4">
Shouldn't the `additionalInfo` be added to a `<Row>` of its own?

To verify that, perhaps it might be beneficial to include some test cases to verify behavior when `warning`/ `error` is passed alongside `additionalInfo`? Additionally, incorporating some VRTs could be helpful. You can find references in the `*.visualroute.tsx` files.

Reply:
<img width="452" alt="Screenshot 2024-01-31 at 15 20 03" src="https://github.com/commercetools/ui-kit/assets/42547802/3854c857-c99b-4d50-a176-f5842cc714a2">

Thats how it looks in the in the vercel deployment. But I can add a VRT if you want

Reply:
Good catch it doesn't support this case with multiline. Will fix that